XenAdmin/ConsoleView/VNCTabView.cs

```diff
@@ -1525,14 +1525,13 @@ private static string GetConsolePath(SshConsole customSshConsole)
 
         private void OpenSshConsoleNotFoundDialog()
         {
-            var configureSshClientButton = new ThreeButtonDialog.TBDButton(Messages.CONFIGURE_SSH_CONSOLE_TITLE, DialogResult.Yes,
-                ThreeButtonDialog.ButtonType.NONE);
+            var configureSshClientButton = new ThreeButtonDialog.TBDButton(Messages.CONFIGURE_SSH_CONSOLE_TITLE,

```

Comment:
There was an issue here that hitting Esc or the control box did not close the dialog. Also, I think it looks a bit awkward to have two positive actions/dialog results on the dialog, hence I suggest using Cancel for the second button.